docs/azure/packages.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Package index for Azure SDK for .NET
+description: Listing of the package names, NuGet links, docs links, and source code links for all libraries in the Azure SDK for .NET.
+ms.custom: azure-sdk-dotnet
+author: camsoper
+ms.author: casoper
+ms.date: 07/29/2020
+---
+
+# Azure SDK for .NET package index
+
+[Libraries using Azure.Core](#libraries-using-azurecore)
+
+[All libraries](#all-libraries)
```

Comment:
These duplicate the right nav menu; could replace them with brief text describing what's in the doc and what it's for.